sdk/storage/storage-file-share/src/Clients.ts

```diff
@@ -2756,6 +2760,78 @@ export class ShareDirectoryClient extends StorageClient {
       span.end();
     }
   }
+
+  /**
+   * Renames a directory.
+   * This API does not support renaming a directory from one share to another, or between storage accounts.
```

Comment:
>from one share to another, or between storage accounts [](http://example.com/codeflow?start=52&length=54)

Really? Could rename work across share and accounts?

Reply:
No, it only supported to rename in the same share..